docs/howto_publish.rst

```diff
@@ -0,0 +1,252 @@
+        "me-south-1": "ami-0827faef233b14a29",
+        "ap-northeast-1": "ami-0d119806827c3af22",
+        "sa-east-1": "ami-07f8dfef0a8855f06",
+        "ap-east-1": "ami-047cb2feb00bfc834",
+        "ca-west-1": "ami-061003b943c2d6be8",
+        "ap-southeast-1": "ami-0a2ca6ffb79999bb5",
+        "ap-southeast-2": "ami-0a74f3afdd309dbf2",
+        "ap-southeast-3": "ami-091f9d0adaa612bfb",
+        "ap-southeast-4": "ami-0ccc7ff1fcaf16948",
+        "us-east-1": "ami-0c470d0e3eaf16e67",
+        "us-east-2": "ami-02a7417ff5d866f4b"
+      }
+    },
+    "images-by-group": {}
+  }
+
+The output shows the published image IDs for each region. Those images are not
+public so can only be used from within the same account.
+
+.. note::
+   The command can be run again without publishing anything new as long as the source path file
+   and the config itself doesn't change.
+
+
+Multiple images
+~~~~~~~~~~~~~~~
+
+It's possible to publish multiple images based on the same VMDK file. The configuration looks like:
+
+.. literalinclude:: config-samples/config-multiple-images.yaml
+   :language: yaml
+
+Running `awspub` now will publish 2 images in 2 different regions.
+
+.. code-block:: shell
+
+  awspub --log-file awspub.log create config.yaml
+  {
+    "images": {
+      "my-custom-image": {
+        "eu-central-1": "ami-0b9ed498e040c69e2"
+      },
+      "my-custom-image-2": {
+        "eu-central-2": "ami-03889118047373658"
+      }
+    },
+    "images-by-group": {}
+  }
+
+Parameter substitution
+~~~~~~~~~~~~~~~~~~~~~~
+
+There are cases where part of the configuration file needs to be dynamic.
+`awspub` supports basic template substitution (based on Pythons `string.Template class <https://docs.python.org/3/library/string.html#template-strings>`_) .
+
+.. literalinclude:: config-samples/config-with-parameters.yaml
+   :language: yaml
+
+In this config, the identifier which will be replaced is `$serial`. The mapping
+that is used to replace an identifier with a actual value is also a yaml file
+which contains a mapping structure (dict in Python) that maps identifiers
+to values:
+
+.. literalinclude:: config-samples/config-with-parameters.yaml.mapping
+   :language: yaml
+
+Using both together now with `awspub`:
+
+.. code-block:: shell
+
+  awspub --log-file awspub.log create config.yaml --config-mapping config-mapping.yaml                
+  {
+    "images": {
+      "my-custom-image-20171022": {
+        "eu-central-1": "ami-0df443d5919e31d1b"
+      }
+    },
+    "images-by-group": {}
+  }
+
+
+Image groups
+~~~~~~~~~~~~
+
+There might be cases were the different commands (e.g. `awspub create` or `awspub publish`)
+should only be applied for a subset of the defined images. That's possible with the `group`
+config option:
+
+.. literalinclude:: config-samples/config-minimal-groups.yaml
+   :language: yaml
+
+Now the different `awspub` commands do have a `--group` parameter to filter which
+images this command should operate on:
```

Comment:
Change this statement to:
Use the --group parameter to filter the images that the `awspub` command should operate on: